ipa-core/src/helpers/mod.rs

```diff
@@ -509,6 +509,20 @@ impl Display for TotalRecords {
 }
 
 impl TotalRecords {
+    pub const ONE: Self = match NonZeroUsize::new(1) {
```

Comment:
wow this is nice, I didn't think you could do it without `unsafe_unchecked`

Reply:
This may not have always worked... or maybe it's a case of not thinking to try the obvious thing :). Bonus points to clippy for not complaining that it could be an unwrap.